lib/mayaUsd/render/vp2RenderDelegate/meshViewportCompute.h

```diff
@@ -42,7 +42,6 @@
 
 // Maya 2020 is missing API necessary for compute support
 // OSX doesn't have OpenGL 4.3 support necessary for compute
-// USD before 20.08 doesn't include some OSD commits we rely on
 #if MAYA_API_VERSION >= 20210000 && !defined(OSMac_)
```

Comment:
I decided to leave this MAYA_API_VERSION check here since it goes with the comment. I asked Ivan about this opengl case and he doesn't know so I just left the code alone.